invoice/src/main/java/org/killbill/billing/invoice/generator/FixedAndRecurringInvoiceItemGenerator.java

```diff
@@ -95,7 +95,7 @@ public InvoiceGeneratorResult generateItems(final ImmutableAccountData account,
 
         final InvoicePruner invoicePruner = new InvoicePruner(existingInvoices);
         final Set<UUID> toBeIgnored = invoicePruner.getFullyRepairedItemsClosure();
-        final AccountItemTree accountItemTree = new AccountItemTree(account.getId(), invoiceId);
+        final AccountItemTree accountItemTree = new AccountItemTree(account.getId(), invoiceId, config.getProrationFixedDays());
```

Comment:
⚠️ We need to call the version of the config API that takes a context - whenever, we offer a version that takes a context, it needs to be used. Unfortunately, we don't have a way to statically check for this, so this requires careful attention in the code review... Could you make sure we *only* use this version for `src/main` files - for tests, it's ok to use the other form.

Reply:
fixed in 6f6206c